widgetastic_manageiq/__init__.py

```diff
@@ -5088,6 +5089,60 @@ class MigrationDashboardStatusCard(AggregateStatusCard):
     ROOT = ParametrizedLocator(
         './/div[contains(@class, "card-pf-aggregate-status")'
         ' and not(contains(@class, "card-pf-aggregate-status-mini")) and'
-        ' h2[contains(@class,"card-pf-title")  and contains(.,{@name|quote})]]'
+        ' h2[contains(@class,"card-pf-title")  and contains(normalize-space(.),{@name|quote})]]'
     )
-    TITLE_ANCHOR = ".//h2"
+    TITLE_ANCHOR = './h2'
+
+    @property
+    def is_active(self):
+        return 'active' in self.root_browser.classes(self.ROOT or self.__locator__)
+
+
+class v2vPaginator(Paginator):
+    """ Represents Paginator control for V2V."""
+
+    PAGINATOR_CTL = './/div[contains(@class,"form-group")][./ul]'
+    './input[contains(@class,"pagination-pf-page")]'
+    CUR_PAGE_CTL = './/span[./span[contains(@class,"pagination-pf-items-current")]]'
+    PAGE_BUTTON_CTL = './/li/a[contains(@title,{})]'
+
+    def next_page(self):
+        self._click_button('Next Page')
+
+    def prev_page(self):
+        self._click_button('Previous Page')
+
+    def last_page(self):
+        self._click_button('Last Page')
+
+    def first_page(self):
+        self._click_button('First Page')
+
+    def page_info(self):
+        return self.browser.text(self.browser.element(self.CUR_PAGE_CTL, parent=self._paginator))
+
+
+class v2vPaginationDropup(SelectorDropdown):
+    ROOT = ParametrizedLocator(
+        './/div[contains(@class, "dropup") and ./button[@{@b_attr}={@b_attr_value|quote}]]')
+
+
+class v2vPaginatorPane(View):
+    """ Represents Paginator Pane for V2V."""
+
+    ROOT = './/form[contains(@class,"content-view-pf-pagination")]'
```

Comment:
Question:
why you need ROOT? as its `View` secondly you are using `widgets` under view.